lms/djangoapps/edxnotes/helpers.py

```diff
@@ -121,6 +121,35 @@ def send_request(user, course_id, page, page_size, path="", text=None):
     return response
 
 
+def delete_all_notes_for_user(user, user_id):
+    """
+    helper method to delete all notes for a user_id, as part of GDPR compliance
+
+    :param user_id: The user object associated with the deleted notes
+    :return: response (requests) object
+
+    Raises:
+        RequestException - when notes api is not found/misconfigured.
+    """
+    # TODO:PLAT-2001 add to master LMS endpoint.
+    url = get_internal_endpoint()
+    headers = {
+        "x-annotator-auth-token": get_edxnotes_id_token(user),
+        "user_id": user_id
```

Comment:
I think `user_id` should be in the body, not the headers, based on https://github.com/edx/edx-notes-api/pull/53/files#diff-146421ea6956f59c984888c749775643R356, but looking at that PR's tests, it seems like the headers also get included as part of `request.data`, which I didn't realize when I reviewed it. In any case, my understanding of headers vs. body is that headers are typically used for auth, metadata, etc. like the auth token and the body is used for data that's needed for "business logic." Sorry if I'm mansplaining REST APIs, I'm not 100% on this.

Reply:
@sanfordstudent I'll buy it. I can switch that around. I'm not even 55% on REST APIs